config/locales/sidebars.en.yml

```diff
@@ -3,6 +3,7 @@ en:
   sidebar_common:
     - &sidebar_fixed_fees Fixed fees
     - &sidebar_graduated_fees Graduated fees
+    - &sidebar_hardship_fees Hardship fees
```

Comment:
🤔 hardship fees are not a kind of fee, from a business perspective, they are merely the same kinds of fees claimed on a hardship claim

Reply:
although line 24 does call them that, in which case that line should change to use the anchor too